action.yml

```diff
@@ -31,6 +31,14 @@ inputs:
   branch:
     description: Sets the branch in which the action will run. Default to `github.ref_name` if not provided
     required: false
+  apiProtocol:
+    description: Optionally uses graphql API. Used in combination with setting setupGitUser to false will give you signed commits.
+    required: false
+    default: 'rest'
+    type: choice
+    options:
+      - rest
+      - graphql
```

Comment:
I guess the behavior right now isn't really `rest`. It doesn't use the API at all.